agent/consul/auto_config_endpoint_test.go

```diff
@@ -173,7 +174,7 @@ func TestAutoConfigInitialConfiguration(t *testing.T) {
 		c.TLSConfig.InternalRPC.VerifyOutgoing = true
 		c.TLSConfig.InternalRPC.VerifyIncoming = true
 		c.TLSConfig.InternalRPC.VerifyServerHostname = true
-		c.TLSConfig.InternalRPC.TLSMinVersion = "tls12"
+		c.TLSConfig.InternalRPC.TLSMinVersion = types.TLSv1_2
```

Comment:
Assuming this test passes, it would indicate a backwards compatibility issue with older client agents running v1.11.x or lower.

Reply:
I see now. This is just the server config and is not setting up a mock response.

Nothing to see here.